### ISSUE TITLE
Get dm seen list

### ISSUE THREAD

**Issue by Androz2091** [austinhuang0131/barinsta#1621]
First of all thank you for this awesome app! ♥ 

I would like to be able to get the list of the users who have seen the messages like on Instagram "seen by x, y, z". Is it already possible or will it be?

**Comment by Deleora** [comment]
Is this a feature of group chats?

**Comment by Androz2091** [comment]
Yes. And it's the same for direct messages to a specific user, we can see whether they have seen the message or not.
